tools/fetch_elfutils.sh

```diff
@@ -68,9 +68,13 @@ patch -p1 < "${SCRIPT_DIR}/elfutils.patch"
 if [[ "$(basename "${C_COMPILER}")" == clang* ]]; then
   export CFLAGS="-Wno-xor-used-as-pow -Wno-gnu-variable-sized-type-not-at-end -Wno-unused-but-set-parameter"
 fi
-export CFLAGS="${CFLAGS-""} ${EXTRA_CFLAGS}"
+# It is important NOT to set CFLAGS if you don't mean to
+# Otherwise you will not benefit from -O2 in the elfutils compilation
+if [ -n "${EXTRA_CFLAGS-""}" ]; then
```

Comment:
With this fix, we still compile elfutils without optimzations for clang (because of extra warning flags set in CFLAGS)

Reply:
Ah yes !